### PR TITLE
LPS-185656 [PRM] Add Due Date email Templates 

### PR DESCRIPTION
**Pull request by luisacmn**

### CODE REVIEW

**Comment by luisacmn** [comment]
ci:test:sf

**Comment by luisacmn** [comment]
ci:test:acceptance

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"description": "",
+	"editorType": "richText",
+	"name": "Claim Due Date Notification Template",
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.json:4]:
```suggestion
	"name": "Claim 1 Day Due Date Notification Template",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[
+	{
+		"active": true,
+		"errorMessage": {
+			"en_US": "Error"
+		},
+		"externalReferenceCode": "CLAIM-DUE-DATE-NOTIFICATION",
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json:7]:
```suggestion
		"externalReferenceCode": "CLAIM-1-DAY-DUE-DATE-NOTIFICATION",
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+		"label": {
+			"en_US": "Notification Due Date Template Action"
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json:9]:
```suggestion
			"en_US": "Claim 1 Day Notification Due Date Template Action"
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+		},
+		"name": "notificationDueDateTemplateAction",
```
Comment (thaynam) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/notification-templates/claim-due-date-1-template/notification-template.object-actions.json:11]:
```suggestion
		"name": "notification1DayDueDateTemplateAction",
```

**Review by thaynam — CHANGES_REQUESTED** [review]
**We need to set email reminder for 15 days, 5 days, 1 day before the Due Date of the MDF Activity.**

@luisacmn segue os padroes sugeridos na PR para nomear as notifications.

**Comment by luisacmn** [comment]
ci:test:sf

**Comment by thaynam** [comment]
[resent](https://github.com/pt-liferay-solutions/liferay-portal/pull/4306)

**Pull request closed without merging**